tests/Common/AbstractMigrationBuilderTest.php

```diff
@@ -31,8 +30,7 @@ protected function setUp(): void
 
     public function testExecute(): void
     {
-        $this->createTable('test', ['id' => 'int']);
-
+        $this->builder->createTable('test', ['id' => $this->builder->integer()]);
```

Comment:
Why repalce usage `yiisoft/db` to builder for prepare database in tests?

Reply:
Because we need to test MigrationBuilder not db.

Reply:
That is why need to use `db` for preapre DB and after test `MigrationBuilder`.

For example test `MigrationBuilder::execute`:

1) Preapre DB with `db`.
2) Run `$this->builder->execute`
3) Check results.

Reply:
[And does it really make any difference?](https://github.com/yiisoft/yii-db-migration/blob/460dd10a45e7ea2daf4862df2c0d9d2a4d3e5d40/src/MigrationBuilder.php#L191)

Reply:
It's not command we're testing. It's tested elsewhere.

So we assume it's correct.

Thus, it's fine to rely on it.

I see no issue.

Reply:
> I see no issue.

Issue is that for preare DB use non-tested code.

Reply:
It is OK in this case. That's very shallow abstraction.